client/packages/inventory/src/Stocktake/DetailView/ContentArea/columns.tsx

```diff
@@ -206,28 +206,32 @@ export const useStocktakeColumns = ({
     {
       key: 'difference',
       label: 'label.difference',
+      Cell: props => (
+        <NumberCell {...props} defaultValue={UNDEFINED_STRING_VALUE} />
+      ),
       align: ColumnAlign.Right,
       sortable: false,
       accessor: ({ rowData }) => {
         if ('lines' in rowData) {
           const { lines } = rowData;
-          const total =
-            lines.reduce(
-              (total, line) =>
-                total +
-                (line.snapshotNumberOfPacks -
-                  (line.countedNumberOfPacks ?? line.snapshotNumberOfPacks)),
-              0
-            ) ?? 0;
-          return (total < 0 ? Math.abs(total) : -total).toString();
-        } else {
-          return (
-            (rowData.countedNumberOfPacks ?? rowData.snapshotNumberOfPacks) -
-            rowData.snapshotNumberOfPacks
-          );
-        }
-      },
-    },
+                const total =
```

Comment:
Looks like more formatting noise... we should check out your config after lunch!